framework/src/main/java/org/checkerframework/framework/stub/AnnotationFileParser.java

```diff
@@ -622,37 +696,55 @@ private void processPackage(PackageDeclaration packDecl) {
     /**
      * Process a type declaration: copy its annotations to {@code #annotationFileAnnos}.
      *
+     * <p>This method stores the declaration's type parameters in {@link #typeParameters}. When
+     * processing an ajava file, where traversal is handled externaly by a visitor, these type
```

Comment:
Does "a visitor" specifically mean `AjavaParserVisitor`?  If possible, be specific.